stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -187,16 +195,11 @@ internal class ChatDomainImpl internal constructor(
         _online.value = false
         _totalUnreadCount.value = 0
         _channelUnreadCount.value = 0
-        _errorEvent.value = null
         _banned.value = false
         _mutedUsers.value = emptyList()
-        if (activeChannelMapImpl != null) {
-            activeChannelMapImpl.clear()
-        }
 
-        if (activeQueryMapImpl != null) {
-            activeQueryMapImpl.clear()
-        }
```

Comment:
We had null checks on non-nullable, final values here, because this code was being called from an `init` block that was higher up in the class than the property declarations, and therefore those were not initialized yet. Moved the properties up top to avoid this.